onsite/lucxor/flr.py

```
@@ -447,29 +434,16 @@ def perform_minorization(self) -> None:
                 continue
 
-            # Initialize arrays
-            x = np.zeros(n)
-            f = np.zeros(n)
-            fcopy = np.zeros(n)
-            forig = np.zeros(n)
+            # Fill data (vectorized)
+            pairs = [minor_map[i] for i in range(n)]
+            x = np.array([p[0] for p in pairs])  # delta scores
+            f = np.array([p[1] for p in pairs])  # FDR values
+            forig = f.copy()
```

Ruff (0.14.10)

Local variable `forig` is assigned to but never used

Remove assignment to unused variable `forig`

(F841)